lib/metadataTypes/TriggeredSendDefinition.js

```diff
@@ -137,7 +123,11 @@ class TriggeredSendDefinition extends MetadataType {
         // remove IsPlatformObject, always has to be 'false'
         delete metadata.IsPlatformObject;
         // folder
-        super.setFolderPath(metadata);
+        try {
+            this.setFolderPath(metadata);
+        } catch {
+            return;
```

Comment:
any reason to change this logic? are we just ignoring errors here?

Reply:
yes. reason is rather simple. by removing the filter on IsPlatformObject we get journey builder TSDs, but we also get shit ton of deleted & hidden stuff that is no longer visible in the GUI.
the CLI was flooded by warnings before I changed that to simply skip these entries.

ideal? no. it's really just a workaround for yet another quirk in SFMC :(

Reply:
i should add that thats also the reason why I changed the other Util.loggers in this method from WARN to VERBOSE. These other 2 cases occur less often and one should be able to see it at least in the logs because those other entries do show up in the GUI. without a folder though, one couldn't see them anywhere anyways.